### PR TITLE
Microsoft.GraphServices add new SDK version for stable swagger spec

### PR DESCRIPTION
**Pull request by amatukmolina**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by amatukmolina** [comment]
The only change between the previous swagger spec and this one is we changed the operation ids, no functional change was made. For that reason the same tests apply to this spec.

**Comment by amatukmolina** [comment]
@ArthurMa1978 Thanks for the guidance, all tests passed after the changes you suggested. Is there something else we need to do or can we continue with the merge?

**Comment by amatukmolina** [comment]
@archerzz @ArcturusZhang Could you please help us merge this PR if it looks good? I think Arthur is the assignee but I see he's OOF until next week. Arthur told us that for a GA version we need to do some extra polishing steps after this gets merged so we want to get started on that as soon as possible. Thanks!

**Comment by carlosov-ms** [comment]
@archerzz @ArcturusZhang @ArthurMa1978 Are there any action items we still have to perform on our side? If not, could you help us by approving and merging this PR? Our GA release date is end of this month and this is a prerequisite, we want to meet the deadline.

Appreciate all the help you can provide.

Thank you

cc @amatukmolina

**Review by ArcturusZhang — APPROVED** [review]

**Comment by archerzz** [comment]
@carlosov-ms could you fix the conflicts? Thanks.

**Comment by carlosov-ms** [comment]
> @carlosov-ms could you fix the conflicts? Thanks.

@amatukmolina was working on them, Alexis could you provide an update to @archerzz please?

Thanks

**Comment by amatukmolina** [comment]
@archerzz Sorry for the delay, I merged the changes and pushed again. I made a new recording of the tests but the pipelines are saying they need to be uploaded to a different repo via the test-proxy command and I don't have access to push to that one. What should I do to fix those errors? Thanks!

File: sdk/graphservices/Azure.ResourceManager.GraphServices/src/autorest.md
Comment (live1206) [inline sdk/graphservices/Azure.ResourceManager.GraphServices/src/autorest.md:11]: remove the tag since it is the default on in readme.md

**Review by live1206 — COMMENTED** [review]
- Update the changelog with new features added in new version 
- Update the version to stable in `Azure.ResourceManager.GraphServices.csproj`
- Remove `--prerelease` from `dotnet add package Azure.ResourceManager.GraphServices --prerelease` in `sdk\graphservices\Azure.ResourceManager.GraphServices\README.md` L25

**Review by live1206 — DISMISSED** [review]
Need to polish the naming before first GA

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**
